scripts/zones/Periqia/npcs/Ancient_Lockbox.lua

```diff
@@ -66,6 +74,26 @@ entity.onTrigger = function(player, npc)
                 { itemid = 0,                    droprate = 500 },
             },
         },
+
+        [xi.assault.mission.SHOOTING_DOWN_THE_BARON] =
+        {
+            {
+                { itemid = xi.item.HI_POTION_P2,   droprate = 850 },
+                { itemid = 0,                       droprate = 150 },
+            },
+            {
+                { itemid = xi.item.HI_POTION_P3,   droprate = 50 },
+                { itemid = 0,                       droprate = 950 },
+            },
+            {
+                { itemid = xi.item.HI_POTION_TANK, droprate = 400 },
+                { itemid = 0,                       droprate = 600 },
+            },
+            {
+                { itemid = xi.item.HI_RERAISER,    droprate = 200 },
+                { itemid = 0,                       droprate = 800 },
+            },
```

Comment:
Could you align this tables?

Also, mark the PR as tested."

Reply:
Done!